Sources/XCTest/Public/XCTestMain.swift

```diff
@@ -62,7 +62,16 @@
 public func XCTMain(_ testCases: [XCTestCaseEntry]) -> Never {
     XCTMain(testCases, arguments: CommandLine.arguments)
 }
+
 public func XCTMain(_ testCases: [XCTestCaseEntry], arguments: [String]) -> Never {
+  XCTMain(testCases, arguments: arguments, observation: PrintObserver())
+}
+
+public func XCTMain(
+  _ testCases: [XCTestCaseEntry],
+  arguments: [String],
+  observation: XCTestObservation
```

Comment:
- Can we make the type of the `observation:` parameter an Array of XCTestObservation elements, so callers can pass multiple observers if needed? I think expanding this API to allow passing _multiple_ observers would make it even more flexible.
- Can we rename `observation:` to `observers:` to better reflect that that it's plural (after changing to Array)?